docs/en/end-user/monitoring.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Monitoring
+---
+
+TBD, Content Overview：
+
+1. Install monitoring trait(along with prometheus/grafana controller).
```

Comment:
Move all installation scripts to a separate doc may be named `Install Capability Providers` (e.g. https://knative.dev/docs/install/install-extensions/)